src/component/Header/HeaderResponsive.scss

```diff
@@ -0,0 +1,18 @@
+//For Tabs
+@media (min-width:768px) and (max-width:990px){}

```

Comment:
no need to specify the min-width in query because from 990 we are already trying to responsive the corresponding sections till our new query so no need for min-width.